mvn_setup/tasks-parent/tasks-ui/src/core/MyWork/MyRecentActivity.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Typography, Paper } from "@mui/material";
+import { useMemo } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { UserActivity } from "taskclient/client-types";
+import { useOrg } from "taskclient/hooks";
+
+const ActivityItem: React.FC<{activity: UserActivity}> = ({ activity }) => {
+  
```

Comment:
This `.useMemo()` is not needed here, because this ActivityItem will be rendered every time, for each item.  The item itself doesn't change -- it's immutable.